package/MDAnalysis/core/AtomGroup.py

```diff
@@ -685,6 +764,7 @@ def universe(self):
             return self._universe
 
     @universe.setter
+    @deprecate(message="{}; Atoms will not be able to leave their Universes.".format(_FIFTEEN_DEPRECATION))
```

Comment:
It used to be that Parser created Atoms, then the Universe assigned itself to each Atom afterwards in init